### DISCUSSION TITLE
dde-file-manager Releases page on Github: add AppImage

### DISCUSSION THREAD

**Discussion by vitaly-zdanevich** [linuxdeepin/developer-center#8761] [General | 常规讨论]
.

**Comment by BLumia** [comment]
TBH, AppImage might make sense for regular applications, but not for a system-level (or say, Desktop Environment-level) component. Any specific reason why you want the file manager as an AppImage package?

**Reply by vitaly-zdanevich** [reply]
I prefer AppImage because it has all dependencies inside. See another file manager with appImage https://github.com/doublecmd/doublecmd/releases/tag/v1.1.14

**Reply by BLumia** [reply]
Just a quick FYI, dde-file-manager is not a pure standalone file manager utility (i.e. not a regular application), but also tightly integrated with DDE (Deepin Desktop Environment). You can consider it as a system (actually, DDE) component, so it's unlikely it will work as a standalone app.

**Reply by vitaly-zdanevich** [reply]
Hmmm, sad :(
